main.go

```diff
@@ -205,14 +217,13 @@ func findMatchingIssue(search []jira.Issue, summary string) *jira.Issue {
 	return nil
 }
 
-func logError(err error, response *jira.Response) {
-	log.Println(err)
-	log.Println(response.StatusCode)
+func logError(e error, response *jira.Response) {
 	all, err := io.ReadAll(response.Body)
+
 	if err != nil {
-		log.Println("Could not read body", err)
+		log.WithError(e).WithField("StatusCode", response.StatusCode).Errorf("Could not read body: %q", err)
 	} else {
-		log.Println(string(all))
+		log.WithError(e).WithField("StatusCode", response.StatusCode).Error(string(all))
```

Comment:
You could extract `log.WithError(e).WithField("StatusCode", response.StatusCode)` into a variable at the beginning of the function, this would make this block easier to read.